tutorial_ja/README.md

```diff
@@ -0,0 +1,196 @@
+
+では、上のコードの行を簡単に解説しておきましょう。上の行はいくつかのことを行っています：まず、変数 `c` が新しいCETEIオブジェクトとして定義されています。このオブジェクトはソースファイルの読み込みとスタイル指定を行なってくれます。次に、`c`にソースファイルを読み込んでHTMLカスタム要素に変換するように指示し、その結果を受け取ってindex.htmlファイルの`<body>`に配置する関数も与えています。`document.getElementsByTagName('body')` は組み込みの `document` オブジェクト (`document` はブラウザに読み込まれた HTML ドキュメント) で利用できる関数を呼び出して、すべての body 要素を見つけて Array (インデックス番号でアクセスできるリスト。Pythonで言うリストとほぼ同じ。) で返しています。body 要素はひとつだけなので、Array の1つ目（0から数えるので0番）にある項目を取得します。その項目であるHTML要素に、先ほど読み込んだTEIドキュメントを子要素として追加しています。
+
+
+＜TEI/XMLファイルをindex.htmlのJavscriptに読み込む＞
+
+この時点で、Atomを使用している場合は、パッケージメニューからHTMLプレビューを実行し、文書を見ることができるようになっているはずです。Atom を使っていない場合は、ドキュメントをウェブサーバーに置いてみるとよいでしょう。GitHub の使用に慣れているなら、GitHub Pages ([tutorial](https://guides.github.com/features/pages/)-テーマのステップをスキップして、読んでいるチュートリアルが入っているリポジトリだけを使うことができます) を使用することができます。Pythonがインストールされていれば、tutorialディレクトリで簡単なWebサーバをコマンドで実行することができます。
+
+```bash
+python -m SimpleHTTPServer
+```
+また、お使いのコンピュータにウェブサーバー機能が組み込まれている場合もありますし、[MAMP](https://www.mamp.info)などをインストールすることもできます。
+
+現時点でのプレビューには様々な不具合があり、それを修正する必要があります。まず、CSSを追加してファイル内のTEI要素を処理することから始め、その後、CETEIceanの動作を追加してより洗練されたことを行うことにしましょう。もしまだXMLソースファイルをご覧になっていなければ、今すぐご覧になって、CETEIceanがすでに行っていることと、行っていないことを確認されるとよいでしょう。そうするとまず、画像付きの図が適切に読み込まれていることに気づくでしょう（画像がウェブサイトにホストされているため、オンラインの場合の話ですが）。teiHeader` のコンテンツは表示されないはずです。ページの始まりや行の始まりは無視されますが、`div`要素や`p`要素はブロックとしてフォーマットされます。少し調べると、ソースファイルの本文には19種類の要素があることがわかります：
+
+ * div
+ * head
+ * p
+ * pb
+ * hi
+ * figure
+ * graphic
+ * lb
+ * sic
+ * opener
+ * dateline
+ * closer
+ * salute
+ * signed
+ * emph
+ * lg
+ * l
+ * foreign
+ * name. 
+ 
+これらの中には、特別なスタイルや動作が必要ないものと、絶対に必要なものがあります：
+
+css/` フォルダにある `tei.css` ファイルを見てみましょう。ご覧のように、今のところ2つのルールしかありません：
+
+```css
+tei-div {
+  display: block;
+}
+tei-p {
+  display: block;
+  margin-top: .5em;
+  margin-bottom: .5em;
+}
+```
+CSSセレクタの要素名には "tei "が付いていますが、これはCETEIceanがTEI要素をHTMLカスタム要素に変換するために行っていることです。これらのルールは、divがブロックとして表示され（新しい行で始まり、最後に改行される）、段落と同様に、後者も前後に多少のスペースを持つことを意味します。現在スタイルが設定されていない要素にどのようなスタイルを適用するかを決めるのは必ずしも容易ではありませんが、まずは簡単なケースをいくつか挙げてみましょう。ソースのXML文書では、目次と索引にリストを使っており、それらには常に `@type="simple"` が付いています。CSSを使って、これらをリストとしてフォーマットすることができます。以下を `tei.css` ファイルに追加してください：
+
+```css
+tei-list[type=simple] {
+  list-style-type: none;
+}
+tei-list[type=simple]>tei-item {
+  display: list-item;
+}
+```
+
+上記のCSSの最初のセレクタ（tei-list[type=simple]の部分）は `<tei-list type="simple">` 要素にマッチし、リストアイテムが箇条書きや数字で装飾されてはならないことを「list-style-type: none」で指定しています。2 番目のセレクタ（tei-list[type=simple]>tei-itemの部分）は `<tei-list>` 要素をリストアイテムとして識別します (HTML の `<li>` と同じです)。この変更を行ってからプレビューを再読み込みすると、目次がリストとしてフォーマットされているのがわかると思います。余白や別のスタイルを追加して、見栄えをよくするために実験することもできます。また、`<hi rend="italics">` の要素も同様にして簡単に扱うことができます：
+```css
+tei-hi[rend=italics] {
+  font-style: italic;
+}
+```
+
+改行（Line beginning）はちょっとややこしく、CSSの「擬似セレクタ」が必要です。
+Line beginnings are a little trickier, and require a CSS "pseudo-selector":
```

Comment:
Same here, was this left intentionally?

Reply:
I'm sorry for my mistake. The English sentences have been deleted.